src/test/java/io/stargate/sgv2/jsonapi/api/v1/HTTPLimitsIntegrationTest.java

```diff
@@ -2,68 +2,68 @@
 
 import static io.restassured.RestAssured.given;
 import static io.stargate.sgv2.common.IntegrationTestUtils.getAuthToken;
-import static org.assertj.core.api.Assertions.assertThat;
 
-import com.fasterxml.jackson.databind.ObjectMapper;
-import com.fasterxml.jackson.databind.json.JsonMapper;
-import com.fasterxml.jackson.databind.node.ArrayNode;
-import com.fasterxml.jackson.databind.node.ObjectNode;
 import io.quarkus.test.common.QuarkusTestResource;
 import io.quarkus.test.junit.QuarkusIntegrationTest;
+import io.restassured.config.HttpClientConfig;
+import io.restassured.config.RestAssuredConfig;
 import io.restassured.http.ContentType;
 import io.stargate.sgv2.api.common.config.constants.HttpConstants;
 import io.stargate.sgv2.common.CqlEnabledIntegrationTestBase;
 import io.stargate.sgv2.jsonapi.testresource.DseTestResource;
-import org.junitpioneer.jupiter.RetryingTest;
+import java.io.InputStream;
+import java.util.Collections;
+import org.apache.http.params.CoreProtocolPNames;
+import org.junit.jupiter.api.Test;
+import org.testcontainers.shaded.org.apache.commons.lang3.RandomUtils;
```

Comment:
This is probably not safe -- should not rely on shaded internal version (maybe just IDE auto-completion caused?)

Reply:
Done.